WORKSPACE

```diff
@@ -350,15 +350,15 @@ gapic_generator_csharp_repositories()
 # Ruby
 ##############################################################################
 
-_gapic_generator_ruby_version = "2b66e7aca8d5d7d4cb7bf436776d7713d264cab8"
+_gapic_generator_ruby_version = "v0.9.2"
 
-_gapic_generator_ruby_sha256 = "b00e2fa2c6f6734a32cffe77ffe5a74d02bbfaa4ad70dd92fde43c47c090b663"
+_gapic_generator_ruby_sha256 = "d4295868bf097a78ac4c2f3afcb7dad540d00a9acb782508ff252be228944809"
 
 http_archive(
     name = "gapic_generator_ruby",
     sha256 = _gapic_generator_ruby_sha256,
-    strip_prefix = "gapic-generator-ruby-%s" % _gapic_generator_ruby_version,
-    urls = ["https://github.com/googleapis/gapic-generator-ruby/archive/%s.tar.gz" % _gapic_generator_ruby_version],
+    strip_prefix = "gapic-generator-ruby-gapic-generator-%s" % _gapic_generator_ruby_version,
```

Comment:
Please clarify in PR description why this got such a weird name (gapic-generator used as both suffix and prefix).